index.html

```diff
@@ -37,24 +42,261 @@ <h1>hey there. <br> I'm Hamza</h1>
+      </div>
+    </section>
+    <!-- End of Projects section -->
+
+    <!-- About me section -->
+    <section class="black-section about-section">
+      <div class="about-header">
+        <div class="section-title about-title">
+          <h2>About me</h2>
+        </div>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button class="btn-primary btn-about" type="button">
+          Get My Resume
+        </button>
+      </div>
+      <div class="image-about">
+        <img src="./image_icons/Frame-about.svg" alt="frame-image" width="200" height="190" />
+      </div>
+      <div class="underline about-under"></div>
+      <div class="skills-container">
+        <ul class="skills-language d-flex">
+          <li>
+            <div class="d-flex skill-title">
+              <img src="./image_icons/Rectangle-inverse.svg" alt="rectangle" />
+              <h3>Languages</h3>
+            </div>
+            <ul>
+              <li><h4>Javascipt</h4></li>
+            </ul>
+            <ul>
+              <li><h4>Ruby</h4></li>
+            </ul>
+            <ul>
+              <li><h4>Html</h4></li>
+            </ul>
+            <ul>
+              <li><h4>Css</h4></li>
+            </ul>
+          </li>
+        </ul>
+        <ul class="skills-language">
+          <li>
+            <div class="d-flex skill-title">
+              <img src="./image_icons/Rectangle.svg" alt="Rectangle" />
+              <h3>Frameworks</h3>
+            </div>
+            <ul>
+              <li><h4>Bootsrap</h4></li>
+            </ul>
+            <ul>
+              <li><h4>Ruby on Rails</h4></li>
+            </ul>
+            <ul>
+              <li><h4>RSPec</h4></li>
+            </ul>
+            <ul>
+              <li><h4>CapyBara</h4></li>
+            </ul>
+            <ul>
+              <li><h4>Selenium</h4></li>
+            </ul>
+          </li>
+        </ul>
+        <ul class="skills-language">
+          <li>
+            <div class="d-flex skill-title">
+              <img src="./image_icons/Rectangle_57.svg" alt="rectangle" />
+              <h3>Skills</h3>
+            </div>
+            <ul>
+              <li><h4>Github</h4></li>
+            </ul>
+            <ul>
+              <li><h4>Codepen</h4></li>
+            </ul>
+            <ul>
+              <li><h4>RSPec</h4></li>
+            </ul>
+            <ul>
+              <li><h4>Gitlab</h4></li>
+            </ul>
+            <ul>
+              <li><h4>Terminal</h4></li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <!-- End of About-me section -->
+    <!-- Contact-me section -->
+    <!--     <section class="contact"></section>-->
+    <!-- End of Contact-me section -->
```

Comment:
- [ ] Please delete unused commented-out code to keep with the best HTML practices.